### PR TITLE
Remove unneeded vendored test files in order to avoid vegeta dep

### PR DESCRIPTION
**Pull request by tcnghia**

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -63,6 +63,7 @@ rm -rf $(find vendor/ -name 'OWNERS')
 # Remove unit tests & e2e tests.
 rm -rf $(find vendor/ -path '*/pkg/*_test.go')
 rm -rf $(find vendor/ -path '*/e2e/*_test.go')
+rm -rf $(find vendor/ -path '*/test/e2e/*.go')
```
Comment (vagababov) [inline hack/update-deps.sh:66]: how about */test/performance and */test/conformance/api ?

Comment (tcnghia) [reply hack/update-deps.sh:66]: We don't pull those into vendor currently

**Pull request merged**